intro_theano/logistic_regression.ipynb

```diff
@@ -413,6 +413,15 @@
     "plt.imshow(plot_data, cmap='Greys', interpolation='none')\n",
     "plt.axis('off')"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": []
```

Comment:
Yes and no.  This adds an empty cell at the end which is the result of doing CTRL-Enter on the last cell.  I tend to leave it in in the notebooks I make.